source/index.html.md

```diff
@@ -963,6 +964,23 @@ Examples:
 }
 ````
 
+### Stop recurring status
+
+On any recurring transaction, issuers can respond with <i>stop recurring</i>,
+withdrawing the consent from the cardholder for a merchant to initiate
+transactions on their behalf.
+
+A <i>stop recurring</i> status is both a decline of the current authorization,
+but also a notice to halt all future recurring transactions on this card from
+this merchant.
+
+If <i>stop recurring</i> responses are not adhered to and recurring
+transactions on this card continues, a fee pr. transaction is incurred by
+the merchant from the card scheme.
+
+Merchants should contact the cardholder to resolve this issue. Only after the
+withdrawal of consent has been lifted, can recurring transactions be resumed.
```

Comment:
I would imagine that the cardholder would have to consent a second time, i.e. create a new first recurring with CSC, 3d-secure and so on?

Reply:
The stop services does not distinguish between different subscriptions.
Getting the customer to agree to a new subscription won't help. The stop-payment blockade will have to be lifted before recurring transaction can be resumed. To quote Visa:

>  **Clearhaus**: What best practices can be pass on to our merchant regarding trying to adhere
to R1 stop requests? On a stop request should merchants freeze all payments,
and contact the cardholder to reaffirm the ones that should continue?
> **Visa**: Yes, exactly!

After the merchant contact the customer/issuer to get the issue resolved, the payments can be resumed. But in reality, this might entail getting the customer to sign-up for at new subscription.